.github/workflows/functions.yml

```diff
@@ -31,6 +31,7 @@ jobs:
     strategy:
       matrix:
         target: [ios, tvos, macos, watchos]
+        swift_version: [5.9, 6.0]
```

Comment:
Do we still need to test on 5.9?

Reply:
It's a good question. I think so since I'm assuming most projects are still using Swift 5.x. I believe we mainly want to test with and without strict Swift concurrency fixes to ensure the strict concurrency related changes don't make the non-strict API difficult to use (in theory, maybe this could never happen?).

In lieu of changing the Swift version, we could change the below build settings, but changing the Swift version in the podspec is simpler to do.

<img width="1643" alt="Screenshot 2025-05-12 at 12 21 26 PM" src="https://github.com/user-attachments/assets/647a1e23-8d13-4963-b0b2-224b1822312c" />

Reply:
I'd prefer to keep testing on Swift 5.9 until I better understand what, if any, test coverage we would miss out on if we tested Swift 6 only.